base/file.jl

```diff
@@ -99,6 +135,17 @@ directory. If the directory already exists, or some intermediate directories do
 this function throws an error. See [`mkpath`](@ref) for a function which creates all
 required intermediate directories.
 Return `path`.
+
+# Examples
+```julia-repl
+julia> mkdir("faketestingdir")
```

Comment:
Whats "fake" with this directory? Perhaps just `testdir` would be enough? (And same for the `mkpath` example below.)